supabase/migrations/20250108075612_everything_else.sql

```diff
@@ -3,7 +3,8 @@
 -- Same as basejump.account_role but to prevent crash in "/usr/local/bin/migra"
 CREATE TYPE public.account_role AS ENUM (
     'owner',
-    'member'
+    'member',
+    'reviewer'
 );
```

Comment:
Really good PR, a minor note is that you rarely modify existing migrations; you create new migrations.